### DISCUSSION TITLE
Calculating the volume of intersections

### DISCUSSION THREAD

**Discussion by oinoirenigonasoindoisanfoidsn** [godotengine/godot-proposals#9691] [3D]
I have a project where I am trying to calculate how much objects are overlapping; I am trying to find the volume of the intersection between two meshes (they don't have collisions. The goal is to be able to make weapons where the more contact it has with the enemy, the more damage it does.

**Comment by AThousandShips** [comment]
This is for discussing ideas for new features for the engine, this sounds more like asking for support for how to do something like that, if you have those kinds of questions please ask them in the [forums](https://forum.godotengine.org/), otherwise please add some more details on what you are wanting to add to the engine
